pkg/constants.go

```diff
@@ -62,3 +62,24 @@ const StatusMaybeDown = 8
 
 // StatusDown is the status value indicating that the monitor is currently down.
 const StatusDown = 9
+
+// AlertContactType is a predefined values from uptimerobot. Need for using AlertContact friendly type.
+const (
+	AlertContactTypePrimaryEmail = 0
+	AlertContactTypeEmail        = 2
```

Comment:
You'll see I've arranged the other constants by _value_ order, not by name—this is so that it's easy to see which values are defined.

Reply:
Fixed